docker/colore/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ruby:2.6.5
+
+RUN apt-get update && apt-get install -y \
+  build-essential \
+  imagemagick \
+  libmagic-dev \
+  libreoffice \
+  tesseract-ocr
+
+# Please keep using version 0.12.3
+# With newer versions of wkhtmltopdf, wkhtmltopdf/wkhtmltopdf#1524 and
+# wkhtmltopdf/wkhtmltopdf#3241 will affect Colore's PDF output
+# TODO: implement PDF comparison specs and update this library
+ARG WKHTMLTOPDF_VERSION=0.12.3
+ARG WKHTMLTOPDF_MD5=6e991e1a1f3293ab673afa015703ef86
+
+RUN wget --quiet https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox-${WKHTMLTOPDF_VERSION}_linux-generic-amd64.tar.xz -O wkhtmltox.tar.xz && \
+    echo "${WKHTMLTOPDF_MD5} wkhtmltox.tar.xz" > MD5SUMS && \
+    md5sum -c MD5SUMS && \
+    tar -xf wkhtmltox.tar.xz && \
+    mv wkhtmltox/bin/wkhtmltopdf /usr/local/bin && \
+    rm -rf wkhtmltox wkhtmltox.tar.xz MD5SUMS
+
+ARG TIKA_VERSION=1.26
+
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-key 184454FAD8697760F3E00D2E4A51A45B944FFD51 && \
+  wget --quiet https://archive.apache.org/dist/tika/tika-app-${TIKA_VERSION}.jar.asc -O tika-app.jar.asc && \
+  wget --quiet https://archive.apache.org/dist/tika/tika-app-${TIKA_VERSION}.jar -O tika-app.jar && \
+  gpg --verify tika-app.jar.asc tika-app.jar && \
+  rm -f tika-app.jar.asc && \
+  mv tika-app.jar /usr/share/java/tika-app.jar && \
+  echo '#!/bin/sh\n\nARGS="$@"\n\n[ $# -eq 0 ] && ARGS="--help"\n\nexec java -jar /usr/share/java/tika-app.jar $ARGS\n' > /usr/local/bin/tika && \
+  chmod +x /usr/local/bin/tika
+
+ENV APP_DIR /colore
+
+EXPOSE 9240
+
+WORKDIR $APP_DIR
+
+# Create application user and add permissions
+RUN groupadd --gid 1000 colore && \
+    useradd --uid 1000 --gid colore --shell /bin/bash --create-home colore && \
+    chown -R colore:colore $APP_DIR && \
+    chmod -R 755 $APP_DIR
+
+# Switch to application user
+USER colore
```

Comment:
I read that it is not good to run Docker applications as root. Any improvement of this is appreciated

https://medium.com/@mccode/processes-in-containers-should-not-run-as-root-2feae3f0df3b